### ISSUE TITLE
[$400,$200] Add TC button to Issues list screen

### ISSUE THREAD

**Issue by cwdcwd** [cloudspokes/GLIB-ChromeExt#10]
### Description:
This is an expansion on the [GLIB-GITHUB-TOPCODER CHROME EXTENSION Challenge](https://www.topcoder.com/challenge-details/30052923/?type=develop&noncache=true). 

### Requirements:
- Add the TC button to Issues list screen 
- Launch challenges using the existing capabilities of the extension on all checked items on the screen
- Log creation errors as comments on the issue tickets and report any other errors as a pop up to the user

![image](https://cloud.githubusercontent.com/assets/1504607/13547841/517ca61c-e2af-11e5-9579-e233572ed096.png)

### Setup 
- fork [this repository](https://github.com/cloudspokes/GLIB-ChromeExt) and work off [this branch](https://github.com/cloudspokes/GLIB-ChromeExt/tree/30053158-multilaunch)

### Submissions:
Upload documentation for how to run your submission
Upload all your source code as a zip
Winner will be required to submit a pull request against [the branch specified](https://github.com/cloudspokes/GLIB-ChromeExt/tree/30053158-multilaunch) !

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Failed to create challenge
Status code: 404
Response body:
```
"<!DOCTYPE html>\n<html>\n  <head>\n    <title></title>\n    <link rel='stylesheet' href='/stylesheets/style.css' />\n  </head>\n  <body>\n    <h1>Not Found</h1>\n<h2>404</h2>\n<pre>Error: Not Found\n    at /app/app.js:30:15\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)\n    at trim_prefix (/app/node_modules/express/lib/router/index.js:302:13)\n    at /app/node_modules/express/lib/router/index.js:270:7\n    at Function.proto.process_params (/app/node_modules/express/lib/router/index.js:321:12)\n    at next (/app/node_modules/express/lib/router/index.js:261:10)\n    at /app/node_modules/express/lib/router/index.js:603:15\n    at next (/app/node_modules/express/lib/router/index.js:246:14)\n    at Function.proto.handle (/app/node_modules/express/lib/router/index.js:166:3)\n    at router (/app/node_modules/express/lib/router/index.js:35:12)</pre>\n\n  </body>\n</html>\n"
```

**Comment by cwdcwd** [comment]
Failed to create challenge
Status code: 404
Response body:
```
"<!DOCTYPE html>\n<html>\n  <head>\n    <title></title>\n    <link rel='stylesheet' href='/stylesheets/style.css' />\n  </head>\n  <body>\n    <h1>Not Found</h1>\n<h2>404</h2>\n<pre>Error: Not Found\n    at /app/app.js:30:15\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)\n    at trim_prefix (/app/node_modules/express/lib/router/index.js:302:13)\n    at /app/node_modules/express/lib/router/index.js:270:7\n    at Function.proto.process_params (/app/node_modules/express/lib/router/index.js:321:12)\n    at next (/app/node_modules/express/lib/router/index.js:261:10)\n    at /app/node_modules/express/lib/router/index.js:603:15\n    at next (/app/node_modules/express/lib/router/index.js:246:14)\n    at Function.proto.handle (/app/node_modules/express/lib/router/index.js:166:3)\n    at router (/app/node_modules/express/lib/router/index.js:35:12)</pre>\n\n  </body>\n</html>\n"
```

**Comment by cwdcwd** [comment]
Failed to create challenge
Status code: 404
Response body:
```
"<!DOCTYPE html>\n<html>\n  <head>\n    <title></title>\n    <link rel='stylesheet' href='/stylesheets/style.css' />\n  </head>\n  <body>\n    <h1>Not Found</h1>\n<h2>404</h2>\n<pre>Error: Not Found\n    at /app/app.js:30:15\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)\n    at trim_prefix (/app/node_modules/express/lib/router/index.js:302:13)\n    at /app/node_modules/express/lib/router/index.js:270:7\n    at Function.proto.process_params (/app/node_modules/express/lib/router/index.js:321:12)\n    at next (/app/node_modules/express/lib/router/index.js:261:10)\n    at /app/node_modules/express/lib/router/index.js:603:15\n    at next (/app/node_modules/express/lib/router/index.js:246:14)\n    at Function.proto.handle (/app/node_modules/express/lib/router/index.js:166:3)\n    at router (/app/node_modules/express/lib/router/index.js:35:12)</pre>\n\n  </body>\n</html>\n"
```

**Comment by cwdcwd** [comment]
Failed to create challenge
Status code: 404
Response body:
```
"<!DOCTYPE html>\n<html>\n  <head>\n    <title></title>\n    <link rel='stylesheet' href='/stylesheets/style.css' />\n  </head>\n  <body>\n    <h1>Not Found</h1>\n<h2>404</h2>\n<pre>Error: Not Found\n    at /app/app.js:30:15\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)\n    at trim_prefix (/app/node_modules/express/lib/router/index.js:302:13)\n    at /app/node_modules/express/lib/router/index.js:270:7\n    at Function.proto.process_params (/app/node_modules/express/lib/router/index.js:321:12)\n    at next (/app/node_modules/express/lib/router/index.js:261:10)\n    at /app/node_modules/express/lib/router/index.js:603:15\n    at next (/app/node_modules/express/lib/router/index.js:246:14)\n    at Function.proto.handle (/app/node_modules/express/lib/router/index.js:166:3)\n    at router (/app/node_modules/express/lib/router/index.js:35:12)</pre>\n\n  </body>\n</html>\n"
```

**Comment by cwdcwd** [comment]
Failed to create challenge
Status code: 404
Response body:
```
"<!DOCTYPE html>\n<html>\n  <head>\n    <title></title>\n    <link rel='stylesheet' href='/stylesheets/style.css' />\n  </head>\n  <body>\n    <h1>Not Found</h1>\n<h2>404</h2>\n<pre>Error: Not Found\n    at /app/app.js:30:15\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)\n    at trim_prefix (/app/node_modules/express/lib/router/index.js:302:13)\n    at /app/node_modules/express/lib/router/index.js:270:7\n    at Function.proto.process_params (/app/node_modules/express/lib/router/index.js:321:12)\n    at next (/app/node_modules/express/lib/router/index.js:261:10)\n    at /app/node_modules/express/lib/router/index.js:603:15\n    at next (/app/node_modules/express/lib/router/index.js:246:14)\n    at Function.proto.handle (/app/node_modules/express/lib/router/index.js:166:3)\n    at router (/app/node_modules/express/lib/router/index.js:35:12)</pre>\n\n  </body>\n</html>\n"
```

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Failed to create challenge
Response body:
```
{
    "errorMessage": "[object Object]",
    "challengeURL": "https://www.topcoder.com/challenge-details/undefined/?type=develop&noncache=true"
}
```

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by 852719** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Failed to create challenge
Response body:
```
{
    "id": 30049571,
    "status": "Draft",
    "forumUrl": "https://apps.topcoder-dev.com/forums/?module=Category&categoryID=28502",
    "challengeURL": "https://www.topcoder.com/challenge-details/30049571/?type=develop&noncache=true"
}
```

**Comment by cwdcwd** [comment]
Failed to create challenge
Response body:
```
{
    "id": 30049572,
    "status": "Draft",
    "forumUrl": "https://apps.topcoder-dev.com/forums/?module=Category&categoryID=28503",
    "challengeURL": "https://www.topcoder.com/challenge-details/30049572/?type=develop&noncache=true"
}
```

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder.com/challenge-details/30049574/?type=develop&noncache=true

**Comment by huangxin1** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder.com/challenge-details/undefined/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder.com/challenge-details/30049576/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Status:undefined
Challenge Url: https://www.topcoder.com/challenge-details/30049582/?type=develop&noncache=true
Forum Url: https://apps.topcoder-dev.com/forums/?module=Category&categoryID=28513

**Comment by cwdcwd** [comment]
Challenge created successfully
Status:Draft
Challenge Url: https://www.topcoder.com/challenge-details/30049583/?type=develop&noncache=true
Forum Url: https://apps.topcoder-dev.com/forums/?module=Category&categoryID=28514

**Comment by cwdcwd** [comment]
Challenge created successfully
Status:Draft
Challenge Url: https://www.topcoder-dev.com/challenge-details/30049584/?type=develop&noncache=true
Forum Url: https://apps.topcoder-dev.com/forums/?module=Category&categoryID=28515

**Comment by bwalles** [comment]
![tc-button](https://cloud.githubusercontent.com/assets/17499676/13359374/89d0e73a-dc79-11e5-81ff-d55fcd483c7a.png)

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: http://topcoder.com

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/undefined/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30049911/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30049914/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30049976/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30049984/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30049986/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30050054/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
Challenge created successfully
Challenge Url: https://www.topcoder-dev.com/challenge-details/30050055/?type=develop&noncache=true

**Comment by cwdcwd** [comment]
resolved on #20
